spec/dummy/db/schema.rb

```diff
@@ -9,103 +8,54 @@
 # from scratch. The latter is a flawed and unsustainable approach (the more migrations
 # you'll amass, the slower it'll run and the greater likelihood for issues).
 #
-# It's strongly recommended to check this file into your version control system.
+# It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(:version => 20130124193419) do
+ActiveRecord::Schema.define(version: 20130123233239) do
 
-  create_table "active_admin_comments", :force => true do |t|
-    t.string   "resource_id",   :null => false
-    t.string   "resource_type", :null => false
-    t.integer  "author_id"
-    t.string   "author_type"
-    t.text     "body"
-    t.datetime "created_at",    :null => false
-    t.datetime "updated_at",    :null => false
-    t.string   "namespace"
-  end
-
-  add_index "active_admin_comments", ["author_type", "author_id"], :name => "index_active_admin_comments_on_author_type_and_author_id"
-  add_index "active_admin_comments", ["namespace"], :name => "index_active_admin_comments_on_namespace"
-  add_index "active_admin_comments", ["resource_type", "resource_id"], :name => "index_admin_notes_on_resource_type_and_resource_id"
-
-  create_table "admin_users", :force => true do |t|
-    t.string   "email",                  :default => "", :null => false
-    t.string   "encrypted_password",     :default => "", :null => false
-    t.string   "reset_password_token"
-    t.datetime "reset_password_sent_at"
-    t.datetime "remember_created_at"
-    t.integer  "sign_in_count",          :default => 0
-    t.datetime "current_sign_in_at"
-    t.datetime "last_sign_in_at"
-    t.string   "current_sign_in_ip"
-    t.string   "last_sign_in_ip"
-    t.datetime "created_at",                             :null => false
-    t.datetime "updated_at",                             :null => false
-  end
-
-  add_index "admin_users", ["email"], :name => "index_admin_users_on_email", :unique => true
-  add_index "admin_users", ["reset_password_token"], :name => "index_admin_users_on_reset_password_token", :unique => true
-
-  create_table "questionable_answers", :force => true do |t|
+  create_table "questionable_answers", force: :cascade do |t|
     t.integer  "user_id"
     t.integer  "assignment_id"
     t.integer  "option_id"
-    t.string   "message"
-    t.datetime "created_at",    :null => false
-    t.datetime "updated_at",    :null => false
+    t.string   "message",       limit: 255
```

Comment:
can u minimize the diff for this file?

So get rid of the extra `limit: 255` and adjust spacing accordingly.

Reply:
@ascoppa Is it ok to manually edit a schema, even if it is just eliminating a diff?

Reply:
Ideally we shouldn't have to do this. But ....  my guess is that we are all running different PG versions and that's causing "random" changes to appear on the schema.rb file every time someone runs migrations. 

https://stackoverflow.com/questions/45494084/random-changes-in-schema-rb

Reply:
@ascoppa Ok, so to confirm, the request is all places where `limit: 255` is added?

Reply:
Yes and there's also some extra spacing going on. The idea is to have the minimal diff possible.